go.mod

```diff
@@ -76,6 +75,8 @@ require (
 	google.golang.org/protobuf v1.31.0
 )
 
+require github.com/hashicorp/golang-lru v1.0.2 // indirect
```

Comment:
Idk why it happens for you(you aren't the first one I do a review where this artifact happens), I never saw it myself.

Reply:
I think it is fine:
```
❯ go mod why github.com/hashicorp/golang-lru
# github.com/hashicorp/golang-lru
github.com/celestiaorg/celestia-node/cmd/cel-shed
github.com/celestiaorg/go-header/store
github.com/hashicorp/golang-lru

```

Reply:
I mean the import in its own `require` instead of being grouped with others